Algorithms_and_Data_Structures/Advanced Algorithms/Kosaraju_SCC/kosaraju_scc.py

```
@@ -0,0 +1,495 @@
+"""
+Kosaraju's Algorithm for Strongly Connected Components
+
+This module implements Kosaraju's algorithm for finding strongly connected components
+in a directed graph using two depth-first searches.
+
+Author: Algorithm Implementation
+Date: 2024
+"""
+
+import time
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)